bundles/framework/geometrycutter/BundleModule.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @class Oskari.mapframework.ui.module.common.geometrycutter.BundleModule
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.geometrycutter.BundleModule', function () {
+    this.sandbox = null;
+}, {
+    getName: function () {
+        return this.__name
```

Comment:
I can't find definition for __name here.  This functions as a "basemodule" to reduce boilerplate-code in actual instance.js?